src/jit.h

```diff
@@ -51,6 +51,24 @@ void jit_translate(riscv_t *rv, block_t *block);
 typedef void (*exec_block_func_t)(riscv_t *rv, uintptr_t);
 
 #if RV32_HAS(T2C)
-void t2c_compile(block_t *block, uint64_t mem_base);
+void t2c_compile(riscv_t *, block_t *);
 typedef void (*exec_t2c_func_t)(riscv_t *);
+
+#if RV32_HAS(JIT_CACHE)
+/* The jit-cache records the program counters and the entries of executable
+ * instructions generated by T2C. Like hardware cache, the old jit-cache will be
+ * overridden by the new one which uses the same slot.
+ */
+#define MAX_JIT_CACHE_TABLE_ENTRIES (1 << 12)
```

Comment:
How about `N_JIT_CACHE_ENTRIES`, which stands for the number of JIT cache entries? You should add a statement that the value must be power of 2.